src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php

```diff
@@ -96,15 +96,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $env = $this->getContainer()->getParameter('kernel.environment');
+        $address = $input->getArgument('address');
+
+        if (false === strpos($address, ':')) {
+            $output->writeln('The address has to be of the form <comment>bind-address:port</comment>.');
```

Comment:
This is the same message that returns `server:start`. You are right, isn't very clear, the built-in webserver command help show the format as `addr:port`. What do you think?